packages/collector/test/tracing/frameworks/bookshelf/test.js

```diff
@@ -15,7 +15,14 @@ const globalAgent = require('../../../globalAgent');
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
 
-mochaSuiteFn('frameworks/bookshelf', function () {
+// Bookshelf wasn't updated since 2020.
+// Waiting for https://github.com/bookshelf/bookshelf/pull/2125
+// We currently can't install bookshelf, because
+// it requires knex<0.22 and knex<0.22 requires mssqlv6
+// if we install mssql@6 into the collector workspace, typeorm will complain
+// that it needs mssql@9. If we install typeorm into the root, typeorm
+// will complaint that it needs mongodb@5.
+mochaSuiteFn.skip('frameworks/bookshelf', function () {
```

Comment:
FYI: ^

Reply:
I am not sure how to deal with that in the future.

Reply:
Me neither but IMO it's fine if we need to let go of this test. We do not have specific instrumentation for bookshelf anyway, and the test is mostly just for making sure our database driver level instrumentation also works when used via bookshelf.

Reply:
Ok I will remove the whole test for now 👍